content/docs/getting-started/runtime.md

```diff
@@ -28,7 +28,7 @@ runtime or no work will be performed.
 
 One of the unique aspects of Tokio is that futures can be spawned on the runtime
 from within other async tasks. Tasks are the application’s “unit of logic”.
-They are similar to [Go’s goroutine] and [Erlang’s process], but asynchronous.
+They are similar to [Go’s goroutine](https://gobyexample.com/goroutines) and [Erlang’s process](http://erlang.org/doc/reference_manual/processes.html), but asynchronous.
```

Comment:
I think these should be defined at the bottom of the file like other links for consistency.

Reply:
also this overlaps with github.com/tokio-rs/website/pull/391 -- would be great if someone made an isolated PR that just fixed these links so we can avoid merge craziness!

Reply:
Hey @ultrasaurus, I can probably just remove my link changes from here as they are fixed by the other PR.